src/resources/breaking-changes.md

```diff
@@ -0,0 +1,788 @@
+---
+title: Breaking changes
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists all the breaking changes in Dart, organized by release and area.
+
+According to our [breaking change policy], each listed item contains a link to more
+information on the change, such as migration guidance, background, reasoning,
+caveats, and more.
+
+{{site.alert.note}}
+Changes prior to Dart 3.1.0 were not subject to our updated breaking change policy,
+so might not have links to additional information. More details per change might
```

Comment:
This seems a bit off, I would expect one of the following:

- So we might not...
- Additional information may be missing...

I don't know but it feels a bit weird.

Reply:
Yeah, it's kind of an awkward note overall. I rewrote it to get rid of that clause entireIy, but I was also looking for feedback on whether this is even worth mentioning. I included it because I thought if anyone's digging through old breaking changes they might be confused, but I could also just leave it out.